File: src/codes/Users_Manual/phase_corr.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Utils;
~using var autd = Controller.Builder([new AUTD3(Point3.Origin)]).Open(Nop.Builder());
autd.Send(new PhaseCorrection(_dev => _tr => new Phase(0x00)));

File: src/codes/Users_Manual/phase_corr.rs
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() -> Result<(), Box<dyn std::error::Error>> {
# let mut autd = Controller::builder([AUTD3::new(Point3::origin())]).open(autd3::link::Nop::builder())?;
autd.send(PhaseCorrection::new(|_dev| |_tr| Phase::new(0x00)))?;
# Ok(())
# }